### PR TITLE
Enhancement/pure uws build

### PR DESCRIPTION
**Pull request by yasserf**

### CODE REVIEW

File: src/message/connection-endpoint.js [hunk 1]
```diff
@@ -57,13 +57,15 @@ var ConnectionEndpoint = function( options, readyCallback ) {
...
+			console.log( process.versions )
 			const uws = req( fileUtils.lookupLibRequirePath( 'uws' ) );
 
-			this._ws = new ( uws.Server ) ({
+			this._ws = new uws.Server({
 				server: this._server,
 				perMessageDeflate: false
 			} );
 		} catch( e ) {
+			console.log( e )
```
Comment (ronag) [inline src/message/connection-endpoint.js]: debug log?

File: src/message/connection-endpoint.js [hunk 2]
```diff
@@ -57,13 +57,15 @@ var ConnectionEndpoint = function( options, readyCallback ) {
 			// we provide a fake module to patch it if necassary, and look it up
 			// from libs incase of a binary deployment since it's a native module
 			var req = global && global.require ? global.require : require;
+			console.log( process.versions )
```
Comment (ronag) [inline src/message/connection-endpoint.js]: debug log?

Comment (yasserf) [inline src/message/connection-endpoint.js]: Thanks for noticing. This is actually a debug pr I'm using to print stuff on ci systems due to compilation issues

Comment (yasserf) [inline src/message/connection-endpoint.js]: Thanks for noticing. This is actually a debug pr I'm using to print stuff on ci systems due to compilation issues

**Pull request closed without merging**